freshmaker/errata.py

```diff
@@ -379,3 +388,24 @@ def get_pulp_repository_ids(self, errata_id):
         data = self._errata_http_get(
             '/errata/get_pulp_packages/{}.json'.format(errata_id))
         return data.keys()
+
+    def get_cve_affected_srpm_nvrs(self, errata_id):
+        """ Get SRPM nvrs which are affected by the CVEs in errata
+
+        :param errata_id: Errata advisory ID, e.g. 25713.
+        :type errata_id: str or int
+        :return: a list of strings each of them is a source rpm nvr
+        :rtype: list
+        """
+        data = self._errata_rest_get(f"/erratum/{errata_id}/builds_by_cve")
+        srpms = []
+        for cve, data_by_product in data.items():
+            for product, product_data in data_by_product.items():
+                for build in product_data.get("builds", []):
+                    for build_nvr, build_info in build.items():
+                        # module builds are not source rpms, we need SRPMS in variants
```

Comment:
I don't understand what this comment is trying to say. Could you please rephrase it?